### ISSUE TITLE
Add different types of bookmarks

### ISSUE THREAD

**Issue by raucao** [skddc/webmarks#21]
Locations would be nice for starters (maybe just add lat/lng).

**Comment by skddc** [comment]
Thinking about this again, locations would actually be *super* nice for a personal, unhosted Google Maps thing. However, I'm not sure if they should go in the "bookmarks" module or somewhere else. Opinions, anyone?

/cc @galfert @slvrbckt

**Comment by galfert** [comment]
I don't have a strong opinion on this. But the longer I think about this, the more it makes sense to me to have these as bookmarks.

**Comment by skddc** [comment]
So maybe `bookmarks/locations` or `bookmarks/places` or somehing like that?

**Comment by galfert** [comment]
`/bookmarks/locations/` sounds good to me.

**Comment by skddc** [comment]
I have a feeling this issue should go in the bookmarks module repo though. You'd usually bookmark stuff from a map, and ideally using existing POIs, e.g. from OSM data or something. There could be an additional view in Webmarks for that, but I'm not sure the app should include a whole maps service. That's a lot of code and functionality that has nothing to do with bookmarking web pages.

**Comment by silverbucket** [comment]
I agree, should be part of the bookmarks module. I think we do/will be actually be running into group/category types for most modules.

**Comment by raucao** [comment]
Just a note to self (and whoever's interested) about location bookmark folders/sync [from fedi](https://kosmos.social/@raucao/106171659808764306):

> Looks like Plus Codes could be a viable option. Two-digit resolution as folder names perhaps. Then use the other digits in the filenames for higher-resolution sync of details...

> https://plus.codes/8F000000+
